project/LibDependencies.scala

```diff
@@ -0,0 +1,62 @@
+import sbt.{ModuleID, _}
+import uk.gov.hmrc.playcrosscompilation.PlayVersion.{Play25, Play26, Play27}
+import PlayCrossCompilation.dependencies
+
+object LibDependencies {
+
+  lazy val libDependencies: Seq[ModuleID] = dependencies(
+    shared = {
+
+      val playVersion = PlayCrossCompilation.playVersion match {
+        case Play25 => "2.5.19"
+        case Play26 => "2.6.20"
+        case Play27 => "2.7.4"
+      }
+
+      val compile = Seq(
+        "com.typesafe.play" %% "play"            % playVersion,
+        "com.typesafe.play" %% "filters-helpers" % playVersion,
+        "org.joda"          % "joda-convert"     % "2.0.2"
+      )
+
+      val test = Seq(
+        "org.scalatest"     %% "scalatest"  % "3.0.8",
+        "org.pegdown"       % "pegdown"     % "1.6.0",
+        "org.jsoup"         % "jsoup"       % "1.11.3",
+        "com.typesafe.play" %% "play-test"  % playVersion,
+        "org.scalacheck"    %% "scalacheck" % "1.14.0"
+      ).map(_ % Test)
+
+      compile ++ test
+    },
+    play25 = {
+      val test = Seq(
+        "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.1"
+      ).map(_ % Test)
+      test
```

Comment:
just simplify with `play25 = Seq(... % Test)`? etc.

Reply:
Agree, just left it this way for now to not alter their build beyond the minimum